src/Parser/AfflictionWarlock/Modules/WarlockCore/DamageDone.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+
+import {formatThousands, formatNumber } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class DamageDone extends Module {
```

Comment:
I didn't have any logs to test this, but when I logged this.owner.report.friendlyPets, filtered by petOwned === this.owner.playerId, it showed both my pets I was using for the whole raid (Doomguard and Infernal), and since I only needed the IDs I figured out that would be ok.